static/app/views/eventsV2/metricsBaselineContainer.tsx

```diff
@@ -0,0 +1,200 @@
+import {isMultiSeriesStats} from 'sentry/components/charts/utils';
+import {EventsStats, Organization} from 'sentry/types';
+import {getUtcToLocalDateObject} from 'sentry/utils/dates';
+import EventView from 'sentry/utils/discover/eventView';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {useMetricsCardinalityContext} from 'sentry/utils/performance/contexts/metricsCardinality';
+import theme from 'sentry/utils/theme';
+import {transformSeries} from 'sentry/views/dashboardsV2/widgetCard/widgetQueries';
+
+import {SeriesWithOrdering} from '../dashboardsV2/datasetConfig/errorsAndTransactions';
+
+import ResultsChart from './resultsChart';
+import {usesTransactionsDataset} from './utils';
+
+type MetricsBaselineContainerProps = {
+  api: Client;
+  confirmedQuery: boolean;
+  eventView: EventView;
+  location: Location;
+  onAxisChange: (value: string[]) => void;
+  onDisplayChange: (value: string) => void;
+  onIntervalChange: (value: string) => void;
+  onTopEventsChange: (value: string) => void;
+
+  organization: Organization;
+  router: InjectedRouter;
+  // chart footer props
+  total: number | null;
+  yAxis: string[];
+};
+
+export function MetricsBaselineContainer({
+  eventView,
+  yAxis,
+  location,
+  organization,
+  api,
+  ...props
+}: MetricsBaselineContainerProps) {
+  const metricsCardinality = useMetricsCardinalityContext();
+  const displayMode = eventView.getDisplayMode();
+
+  const isRollingOut =
+    organization.features.includes('server-side-sampling') &&
+    organization.features.includes('mep-rollout-flag') &&
+    organization.features.includes('discover-metrics-baseline');
+
+  const disableProcessedBaselineToggle =
+    metricsCardinality.outcome?.forceTransactionsOnly ||
+    displayMode !== 'default' ||
+    !usesTransactionsDataset(eventView, yAxis);
+
+  const apiPayload = eventView.getEventsAPIPayload(location);
+  apiPayload.query = '';
+
+  const globalSelection = eventView.getPageFilters();
+  const start = globalSelection.datetime.start
+    ? getUtcToLocalDateObject(globalSelection.datetime.start)
+    : null;
+
+  const end = globalSelection.datetime.end
+    ? getUtcToLocalDateObject(globalSelection.datetime.end)
+    : null;
+
+  const [showBaseline, setShowBaseline] = useState<boolean>(true);
+  const [metricsCompatible, setMetricsCompatible] = useState<boolean>(false);
+  const [processedLineSeries, setProcessedLineSeries] = useState<
+    LineSeriesOption[] | undefined
+  >(undefined);
+
+  useEffect(() => {
+    let shouldCancelRequest = false;
+
+    if (!isRollingOut || disableProcessedBaselineToggle || !showBaseline) {
+      setProcessedLineSeries(undefined);
+      return undefined;
+    }
+
+    doEventsRequest(api, {
+      organization,
+      partial: true,
+      start,
+      end,
+      yAxis,
+      environment: globalSelection.environments,
+      period: globalSelection.datetime.period,
+      interval: eventView.interval,
```

Comment:
Probably something to look into separately but I noticed that selecting 7d+ left some gaps when hovering with the tooltip. Adding a breakpoint here is giving me back `undefined` for `eventView.interval`


https://user-images.githubusercontent.com/22846452/191074979-4fa97764-2de5-4efa-8037-5a4a486fbed6.mov

Reply:
could perhaps be that intervals are mis-matched, will address in a follow up!

Reply:
Oh sorry just looked at the video and didn't read your comment, ok ty for checking